CMakeLists.txt

```diff
@@ -373,6 +347,41 @@ if (NOT S2N_EXECINFO_AVAILABLE)
 endif()
 feature_probe_result(S2N_STACKTRACE ${S2N_STACKTRACE})
 
+if(S2N_BRANCH_ALIGN AND NOT S2N_FUZZ_TEST)
+    set(BRANCH_ALIGN_FLAG "-Wa,-mbranches-within-32B-boundaries")
+    message(STATUS "Feature probe succeeded: S2N_BRANCH_ALIGN is supported.")
+else()
```

Comment:
Are these status messages necessary? Don't we already print the results of all the feature probes?